src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -349,7 +348,12 @@ static bool USB_HS_PHYCInit(void)
     // Enable PLL internal PHY
     usb_hs_phyc->USB_HS_PHYC_PLL = phyc_pll | USB_HS_PHYC_PLL_PLLEN;
 
-    board_delay(2);
+    /* Original ST code has 2 ms delay for PLL stabilization.
+     * Primitive test shows that more than 10 USB un/replug cycle
+     * showed no error with enumeration
+     * //#include "../../../../hw/bsp//board.h"
```

Comment:
This is called withtin dcd_init() which only happens one when powering on or reset. Plug and unplug shouldn't be affected at all. If it does, it possibly cause the initial failure of the 1st enumeration or something like that. Though the 2ms delay isn't there for no reasons. Since it works for now, we could worry about this later on when there is problem with it :)